include/mlir/Dialect/SPIRV/SPIRVTypes.h

```diff
@@ -73,14 +73,22 @@ class ArrayType : public Type::TypeBase<ArrayType, CompositeType,
                                         detail::ArrayTypeStorage> {
 public:
   using Base::Base;
+  using LayoutInfo = uint64_t;
```

Comment:
Could you put a comment here to mention that if this is 0 it means no layout info?

Reply:
Thanks, updated.